test_erezutils.py

```diff
@@ -87,3 +91,58 @@ def test_chuck_group_1(self):
     def test_chunk_group_2(self):
         data = [1, 2, 3, 4]
         self.assertEqual([[1, 2], [3, 4]], list(chunks(data, 2)))
+
+
+# https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions
+@unittest.skipIf(
+    os.getenv("TRAVIS_PULL_REQUEST") != "false",
```

Comment:
This skips the tests for me locally as well. Can we allow tests to run locally, on master, but not PRs?

Reply:
Yes, that's a bit of a bummer for external contributors, as they won't have access to our S3 so these tests will fail locally.

Reply:
I guess nothing prevents from attempting to reach the bucket with `boto` and skip the tests if the bucket is unreachable?

Reply:
Test reaching the bucket before to run the tests, so that S3 tests are only run when a valid S3 access is detected.